### PR TITLE
Feature/make plotly graphs renderable

### PR DESCRIPTION
**Pull request by LamprosArgy**
Changes in Port Congestion example

### CODE REVIEW

File: docs/examples/jupyter/VoyagesAPI/Port_Congestion_in_China_(Capes).ipynb [hunk 1]
```diff
@@ -0,0 +1,1604 @@
+{
```
Comment (vpanagiotou-signal) [inline docs/examples/jupyter/VoyagesAPI/Port_Congestion_in_China_(Capes).ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    # Port/Areas of interest,
</pre><p>remove last comma</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SignalOceanSdk/SignalSDK/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (vpanagiotou-signal) [inline docs/examples/jupyter/VoyagesAPI/Port_Congestion_in_China_(Capes).ipynb:1]: <p>In the code block below we are merging the oringinal -&gt; original</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SignalOceanSdk/SignalSDK/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (vpanagiotou-signal) [inline docs/examples/jupyter/VoyagesAPI/Port_Congestion_in_China_(Capes).ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Helper 2Function that adds date buttons in time series visualisations
</pre><p>remove 2 before function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SignalOceanSdk/SignalSDK/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (vpanagiotou-signal) [inline docs/examples/jupyter/VoyagesAPI/Port_Congestion_in_China_(Capes).ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.                            center = dict(lat = 0, lon = 180),zoom = 0, mapbox_style = "stamen-terrain",
</pre><p>Change center to <code>center = dict(lat = 0, lon = 0)</code>   in order to centralize the map as is in TSOP.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SignalOceanSdk/SignalSDK/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Comment (LamprosArgy) [reply docs/examples/jupyter/VoyagesAPI/Port_Congestion_in_China_(Capes).ipynb:1]: Done

Comment (LamprosArgy) [reply docs/examples/jupyter/VoyagesAPI/Port_Congestion_in_China_(Capes).ipynb:1]: Done

Comment (LamprosArgy) [reply docs/examples/jupyter/VoyagesAPI/Port_Congestion_in_China_(Capes).ipynb:1]: Done

Comment (LamprosArgy) [reply docs/examples/jupyter/VoyagesAPI/Port_Congestion_in_China_(Capes).ipynb:1]: Done

**Review by vpanagiotou-signal — APPROVED** [review]

**Pull request merged**